trailblazer/store/api.py

```diff
@@ -200,32 +156,15 @@ def update_run_status(self, analysis_id: int, ssh: bool = False) -> None:
         if analysis.workflow_manager == WorkflowManager.TOWER.value:
             self.update_tower_run_status(analysis_id=analysis_id)
         elif analysis.workflow_manager == WorkflowManager.SLURM.value:
-            self.update_analysis_from_slurm_run_status(analysis_id=analysis_id, ssh=ssh)
+            self.update_analysis_from_slurm_run_status(analysis_id=analysis_id)
```

Comment:
do we want to have a controller for this? not sure, up to you, I'm starting to see controllers everywhere lately and I wonder 😬

Reply:
😄 Not for this